LICENSE.txt

```diff
@@ -472,5 +472,48 @@ FAILURE OF THE LIBRARY TO OPERATE WITH ANY OTHER SOFTWARE), EVEN IF
 SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH
 DAMAGES.
 
-   END OF TERMS AND CONDITIONS
-
+                     END OF TERMS AND CONDITIONS
+
+           How to Apply These Terms to Your New Libraries
```

Comment:
[The FSF recommends including it, in fact, you may not be even allowed to remove it at all.](https://www.gnu.org/licenses/gpl-faq.en.html#GPLOmitPreamble)